docs/resources.rst

```diff
@@ -153,3 +153,26 @@ Third-Party Solidity Parsers and Grammars
 
 * `Solidity Parser for JavaScript <https://github.com/solidity-parser/parser>`_
     A Solidity parser for JS built on top of a robust ANTLR4 grammar.
+    
+Blockchain API Providers
+=========================================
+When deploying smart contracts to either the Ethereum testnet (Goerli) or mainnet, most developers prefer to use a blockchain developer platform. These platforms provide not only a suite of tooling and abstractions like SDK and NFT APIs, but also, they provide an API endpoint that acts as a wrapper around the underlying JSON RPC API. This API endpoint makes writing to the blockchain easier and faster. 
```

Comment:
```suggestion
When deploying smart contracts to either the Ethereum testnet (Goerli) or mainnet, most developers prefer to use a blockchain developer platform. These platforms provide not only a suite of tooling and abstractions like SDK and NFT APIs, but also, they provide an API endpoint that acts as a wrapper around the underlying JSON RPC API. This API endpoint makes writing to the blockchain easier and faster.
```

Reply:
Thanks @nikola-matic, noted for the future to make sure the spacing doesn't appear lie this :)